.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - 8
-script: npm test -- --runInBand
```

Comment:
When does the test be executed here?

Reply:
Travis runs `npm test` by default. The `runInBand` flag was to try to mitigate some of the bugs we were seeing with Jest